### ISSUE TITLE
Question: Are future covariates not yet supported in TorchForecastingModel?

### ISSUE THREAD

**Issue by vpozdnyakov** [unit8co/darts#368]
The question is about the `develop` brach. There is a comment in the TorchForecastingModel, `fit` docstring: **Future covariates are not yet supported**. But there is also an example of forecasting using future covariates: https://github.com/unit8co/darts/blob/develop/examples/future-covariates-example.ipynb. Could you please explain how is it possible to forecast using future covariates without fitting using future covariates?

**Comment by pennfranc** [comment]
Thanks a lot for asking. We have indeed recently introduced future covariates, so we have to remove the comment you mentioned in the `fit` dosctring, good catch! However, future covariates in most models are only used when the forecasting horizon exceeds the model's `output_chunk_length`. In that case the model will take its last predictions as well as the next (future) covariates as input to produce more predictions. The exception here is `RNNModel`, which always takes the most recent (future) covariates into account (see its docstring). Sorry if things are a bit confusing right now. We added many new futures in the last few weeks, so the documentation might not be up-to-date everywhere. Again, keep the questions and comments coming, this is very valuable to us!
